src/Bicep.Core/CoreResources.resx

```diff
@@ -1,17 +1,17 @@
-<?xml version="1.0" encoding="utf-8"?>
-<root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Changing all the whitespace, now that's just plain rude.  I'll see if I can fix...